static/js/enketo/widgets/notewidget.js

```diff
@@ -51,7 +51,13 @@ define( function( require, exports, module ) {
     Notewidget.prototype._init = function() {
         var $el = $( this.element );
         var markdownToHtml = angular.element(document.body).injector().get('Markdown').element;
+
+        applyLiveLinkHtml( $el );
+
         markdownToHtml($el.find( '.question-label' ));
+
+        applyLiveLinkEventHandlers( $el );
```

Comment:
I doubt it'd work, as `markdownToHtml` will convert all the elements into text and back again.